lang/cs/Org.Apache.REEF.Client/YARN/IApplicationReport.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
I tried it, if Application has an interface like IApplication, YarnRestClient will throw deserialization error when trying to deserialize data into Application object.